test/events/events.go

```diff
@@ -13,22 +13,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package cloudevent
+package events
 
 import (
 	"fmt"
 	"regexp"
 	"testing"
 	"time"
-
-	"k8s.io/client-go/tools/record"
 )
 
 // CheckEvents checks that the events received by the FakeRecorder are the same as wantEvents,
 // in the same order.
-func CheckEvents(t *testing.T, fr *record.FakeRecorder, testName string, wantEvents []string) error {
+func CheckEvents(t *testing.T, eventChan chan string, testName string, wantEvents []string) error {
```

Comment:
This function could become CheckEventsOrdered and the latter can be CheckEventsUnordered. You can remove references to FakeClient and FakeRecorder in the docstrings as well.

Reply:
:+1: great catch, done.